service/controller/chart/v1/resource/release/update.go

```diff
@@ -37,15 +37,16 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 			}
 		}()
 
-		// TODO: Remove once upgrade --force support for re-installing releases
-		// is supported.
-		//
-		//	See https://github.com/giantswarm/giantswarm/issues/5612
-		//
+		upgradeForce, err := key.HasForceUpgradeAnnotation(cr)
+		if err != nil {
+			return microerror.Mask(err)
+		}
 
 		// We need to pass the ValueOverrides option to make the update process
 		// use the default values and prevent errors on nested values.
-		err = r.helmClient.UpdateReleaseFromTarball(ctx, releaseState.Name, tarballPath, helm.UpdateValueOverrides(releaseState.ValuesYAML))
+		err = r.helmClient.UpdateReleaseFromTarball(ctx, releaseState.Name, tarballPath,
+			helm.UpdateValueOverrides(releaseState.ValuesYAML),
+			helm.UpgradeForce(upgradeForce))
```

Comment:
I still need to test but AIUI we don't need to check the helm release status here. Resources will only be deleted first if the release is unhealthy.

> --force  force resource update through delete/recreate if needed

https://github.com/helm/helm/blob/master/docs/helm/helm_upgrade.md